doc/eventhandler/logginghandler.rst

```diff
@@ -0,0 +1,89 @@
+.. _logginghandler:
+
+Logging Handler Module
+----------------------
+
+.. index:: Logging Handler, Handler Modules
+
+The logging event handler can be used to log the occurrence of an event to the
+python logging facility.
+You can log arbitrary events with a configurable log message, loglevel and
+logger instance. Several tags are available to customize the log message.
+
+The configuration to handle the log messages can be defined in detail with
+the :ref:`advanced_logging`.
+
+Possible Actions
+~~~~~~~~~~~~~~~~
+
+logging
+.......
+
+Emit a log message to the python logging facility when the specified event gets
+triggered (and the conditions match).
+
+**name**
+
+  * *default:* ``pi-eventlogger``
+
+The name of the logger to use when emitting the log message. This can be used
+for a fine-grained control of the log messages via :ref:`advanced_logging`.
+
+.. note:: Logger names beginning with ``privacyidea`` will be handled by the
+   default privacyIDEA logger and will end up in the privacyIDEA log.
+
+**level**
+
+  * *default:* ``INFO``
+
+The log level for the emitted log message. The following levels are available:
+
+  * ``ERROR``
+  * ``WARNING``
+  * ``INFO``
+  * ``DEBUG``
+
+**message**
+
+  * *default:* ``"event={action} triggered"``
+
+The message to send to the logging facility. This message can be customized
+with the following tags:
+
+  * ``{admin}``
```

Comment:
Are these always the same tags as in the notification handler?
Then we could in a new PR/issue move the tags to a document section "event handler tags" and simply add a reference here. 
What do you think?

Reply:
Yes, basically the same but we do not want to include PINs or QR-Codes in the log message.

Reply:
Hm, so the build in the master will succeed.